index.bs

```diff
@@ -2703,6 +2865,7 @@ A <dfn>trigger-registration JSON key</dfn> is one of the following:
 <li>"<dfn><code>aggregatable_source_registration_time</code></dfn>"
 <li>"<dfn><code>aggregatable_trigger_data</code></dfn>"
 <li>"<dfn><code>aggregatable_values</code></dfn>"
+<li>"<dfn><code>aggregate_debug_reporting</code></dfn>"
```

Comment:
Are we intentionally using `aggregate` here even though most of these other fields use `aggregatable`?

Reply:
“aggregate_debug_reporting” seems to read better than “aggregatable_debug_reporting”. I slightly lean towards keeping as is, but don’t feel too strongly. Cc @akashnadan

Reply:
To me the consistency is more important.

Reply:
That's fair, updated to use `aggregatable_debug_reporting`.